Pipfile

```diff
@@ -13,32 +13,35 @@ pytest = "*"
 pytest-asyncio = "*"
 pytest-cov = "*"
 python-dotenv = "*"
-rasterio = "==1.3.8" # same as lambda layer
+rasterio = "*" # same as lambda layer
```

Comment:
This seems strange - are you really intending to remove all these version specifications on the dependencies here and below?  Maybe you can put a comment her or in the pull request about why they aren't needed anymore?

Reply:
We were in old versions of these packages, and adding the most up-to-date titiler with the features we want to use meant updating the dependencies.